Modules/Core/Common/test/itkImageGTest.cxx

```diff
@@ -18,6 +18,7 @@
 
 // First include the header file to be tested:
 #include "itkImage.h"
+#include "itkImageDuplicator.h"
```

Comment:
@blowekamp  Looks like this `#include "itkImageDuplicator.h"` was unnecessary, right?